Feature/random rate of change

alstjq17 opened the pull request:
랜덤변동률

alstjq17 committed:
코인이름에 따른 랜덤변동메소드 작성

alstjq17 committed:
마지막에 )를 빼먹었습니다.

lluke0 commented on server/src/coin/CoinController.java:
```diff
@@ -67,5 +67,18 @@ public void removeCoin(String coinName) {
     }
 
+    // 랜덤 변동률 메소드
+    public void randomRatOfChange(String coinName) {
```
메소드 이름의 경우 변수 + 명사가 적당합니다.
changeCoinRateByRandom이 어떨까요

alstjq17 committed:
변동률은 = 이전가격 - 신가격 / 100
신가격 = 변동률 * 100 + 이전가격
신수량 = 이전수량 - 변동률*이전수량
변동률 = (이전수량 - 신수량) / 이전수량
랜덤 변동률, 가격은 상관관계가 맞지만 랜덤수량은 애매합니다.
